common/net/key_storage.go

```diff
@@ -0,0 +1,112 @@
+package net
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"fmt"
+	"io"
+
+	"golang.org/x/crypto/sha3"
+
+	"github.com/cloudflare/circl/dh/x448"
+)
+
+const GcmTagSize = 16
```

Comment:
@levkoburburas you're right, it should be 128.

Reply:
Sorry, Igor for the misunderstanding, when I was talking about 128 in slack, I meant why exactly 16, there can be anything else, 
I actually. don't know what exact number should be there. To know it I have to review ALTS code deeper. Since it's not my task, but it is time consuming. So if you don't know what is better, let's use `5` as in ALTS

```go
const (
	// Overflow length n in bytes, never encrypt more than 2^(n*8) frames (in
	// each direction).
	overflowLenAES128GCM = 5
)
```